kratos/containers/variables_list_data_value_container.h

```diff
@@ -4,21 +4,16 @@
 //   _|\_\_|  \__,_|\__|\___/ ____/
 //                   Multi-Physics
 //
-//  License:		 BSD License
-//					 Kratos default license: kratos/license.txt
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
 //
 //  Main authors:    Pooyan Dadvand
 //
 //
 
-#if !defined(KRATOS_VARIABLES_LIST_DATA_VALUE_CONTAINER_H_INCLUDED )
-#define KRATOS_VARIABLES_LIST_DATA_VALUE_CONTAINER_H_INCLUDED
+#pragma once
 
 // System includes
-#include <string>
-#include <iostream>
-#include <cstddef>
-#include <cstring>
```

Comment:
Why did you delete the includes? It uses
- `<cstddef>`/ `<cstring>` (line 76)
- `<string>` (line 722) 
- `<iostream>` (line 976)  

By removing it from here you are creating a include dependency on any of the includes below...

Reply:
But `global_variables.h` could probably go away...

Reply:
Reverting